lib/process-services/src/lib/process-list/services/process-filter.service.ts

```diff
@@ -190,7 +191,7 @@ export class ProcessFilterService {
      * @param appId ID of the target app
      * @returns List of filter details
      */
-    callApiProcessFilters(appId?: number) {
+    callApiProcessFilters(appId?: number): Promise<ResultListDataRepresentationUserProcessInstanceFilterRepresentation> {
```

Comment:
Out of curiosity, how does this change addresses Angular 9 compatibility?

Reply:
Hi @DenysVuika ! This specific change probably does not affect ngcc but it seemed like it would be a sensible fix anyway, since the import guessed by TS is deep and across the boundaries of your entry-points. :-)

But for the other changes...

We have a validation repository where we are testing ngcc (our Angular library compatibility compiler) against the top Angular libraries: https://github.com/angular/ngcc-validation.
In this repo the adf-core library was failing to build with ngcc/ngtsc for two reasons. The first is the `SafeStyle` import and the second is the `DOCUMENT` import. 

You can see the failing CI test here https://circleci.com/gh/angular/ngcc-validation/867#tests/containers/2. (Open up the `yarn build` section and search for `adf`).

If you would like us to test your library more completely, you could help us by adding usage of your components, and e2e tests in this test project: https://github.com/angular/ngcc-validation/tree/master/projects/alfrescoadf-core-ngcc

Reply:
I think the reason is failing is because for some reason Travis is not able to pick the environment variables for external PR that are needed in the e2e .
But the unit test are ok

Reply:
> If you would like us to test your library more completely, you could help us by adding usage of your components, and e2e tests in this test project: https://github.com/angular/ngcc-validation/tree/master/projects/alfrescoadf-core-ngcc

There is a way to point from there our e2e? Because a problem that I see if we copy some of our e2e there is that they can easily go out of sync

Reply:
I don't think that we would want an exhaustive set of e2e, since this repo already takes hours to run its CI. One option could be to create a branch in your own repo and update the package versions to use the `@next` versions of Angular, then try running your e2e tests on your own Travis. If there are failures you could raise an issue with us on the Angular github.

Reply:
this looks like a better idea, I will work on it. Thanks for the suggestion

Reply:
Let me know if you have problems setting it up.